packages/wonder-stuff-sentry/src/__tests__/kind-sentry-error.test.js

```diff
@@ -0,0 +1,429 @@
+        });
+
+        it("should return the sentry data of the instance if there are no KindSentryErrors nested in the cause", () => {
+            // Arrange
+            const cause = new Error(
+                "I am not a KindError nor a KindSentryError",
+            );
+            const error = new KindSentryError("MESSAGE", "KIND", {
+                cause,
+                sentryData: {
+                    fingerprint: ["a", "b", "c"],
+                },
+            });
+
+            // Act
+            const result = error.getAllSentryData();
+
+            // Assert
+            expect(result).toIncludeAllMembers([
+                {
+                    tags: {},
+                    contexts: {},
+                    fingerprint: ["a", "b", "c"],
+                },
+            ]);
+        });
+
+        it("should return a stack with SentryData from the error and all nested KindSentryErrors, skipping KindErrors", () => {
+            // Arrange
+            const cause3 = new KindSentryError("MESSAGE3", "KIND3", {
+                sentryData: {
+                    tags: {
+                        tag1: "tagvalue1",
+                    },
+                },
+            });
+            const cause2 = new KindSentryError("MESSAGE2", "KIND2", {
+                cause: cause3,
+                sentryData: {
+                    fingerprint: ["a", "b", "c"],
+                },
+            });
+            const cause1 = new KindError("MESSAGE1", "KIND1", {
+                cause: cause2,
+                metadata: {
+                    sentry: "NOT REAL SENTRY DATA, SHOULD NOT BE IN RESULT",
+                },
+            });
+            const error = new KindSentryError("MESSAGE", "KIND", {
+                cause: cause1,
+                sentryData: {
+                    contexts: {
+                        topOfTheStack: {
+                            hello: "I'm the root error",
+                        },
+                    },
+                },
+            });
+
+            // Act
+            const result = error.getAllSentryData();
+
+            // Assert
+            expect(result).toStrictEqual([
+                {
+                    tags: {},
+                    contexts: {
+                        topOfTheStack: {
+                            hello: "I'm the root error",
+                        },
+                    },
+                    fingerprint: [],
+                },
+                {
+                    tags: {},
+                    contexts: {},
+                    fingerprint: ["a", "b", "c"],
+                },
+                {
+                    tags: {
+                        tag1: "tagvalue1",
+                    },
+                    contexts: {},
+                    fingerprint: [],
+                },
+            ]);
+        });
+    });
+
+    describe("#getMergedSentryData", () => {
+        it("should reduce sentry data stack from bottom to top", () => {
```

Comment:
I'm not sure how important it is to test this implementation detail as long as the output is correct.  No action req'd.